app.rb

```diff
@@ -1,33 +1,57 @@
 require "sinatra/base"
+require "sinatra/reloader"
 
-require_relative "lib/warehouse"
-require_relative "lib/basket"
+Dir["./lib/**/*.rb"].each{|file| require file}
 
-$basket = Basket.new
-$warehouse = Warehouse.new
+module Store
 
-class App < Sinatra::Base
+  class App < Sinatra::Base
+    whouse = CreateWarehouse.new.call
+    basket = CreateBasket.new.call
 
-  get "/" do
-    erb :index
-  end
+    get "/" do
+      @wh_products = FetchProductsFromWarehouse.new.call(whouse.id)
```

Comment:
it isn't good to use variable with single or two characters. `wh` means nothing so it will be better to change it to some english work, and the same in line below